README.md

```diff
@@ -12,14 +12,25 @@ Usage:
  -  Drag and drop a ROM file onto smozROM.exe and follow the instructions
     the program gives you.
 
- -  Please read "NOSE update.html" and "Player Rating Guide.html"
```

Comment:
I think Player Rating Guide is important to keep including, for guiding players on appropriate ratings that keep the original feel (in particular if they use the updated checking formula, they may inadvertently make checking too overpowered or too weak by not understanding the formula)

Reply:
Since we'll get rid of the NOSE update, I think I'll move some of the info from the "NOSE Update.html" file to the "Player Rating Guide.html".

So I think it's important to keep something in README.md that points them to that file.